src/Mvc/test/WebSites/BasicWebSite/StartupWithoutEndpointRouting.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using BasicWebSite.Services;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Abstractions;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace BasicWebSite
+{
+    public class StartupWithoutEndpointRouting
+    {
+        // Set up application services
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddSingleton(new TestService { Message = "true" });
+
+            services.AddAuthentication()
+                .AddScheme<AuthenticationSchemeOptions, BasicAuthenticationHandler>("Api", _ => { });
+            services.AddTransient<IAuthorizationHandler, ManagerHandler>();
+
+            services
+                .AddMvc(options =>
+                {
+                    options.Conventions.Add(new ApplicationDescription("This is a basic website."));
+                    // Filter that records a value in HttpContext.Items
+                    options.Filters.Add(new TraceResourceFilter());
+
+                    // Remove when all URL generation tests are passing - https://github.com/aspnet/Routing/issues/590
```

Comment:
Remove

Reply:
@JamesNK  - I'm not totally sure what you're asking for. Are you suggesting we remove this startup entirely? Currently we test both old and endpoint routing here.

Reply:
Remove the comment

Reply:
OK dope. Wanna smash that like button?